pages/common/tcsh.md

```diff
@@ -0,0 +1,28 @@
+# tcsh
+
+> C shell with file name completion and command line editing.
+> More information: <https://linux.die.net/man/1/tcsh>.
+
+- Start an interactive shell session:
+
+`tcsh`
+
+- Start an interactive shell session without loading startup configs:
+
+`tcsh -f`
+
+- Execute a [c]ommand and then exit:
```

Comment:
```suggestion
- Execute a [c]ommand:
```

Reply:
I use "and then exit" for more verbosity. :)

Reply:
Ok yes, but I err on the side of eliminating redundant verbosity. The word `interactive` tells you what is interactive and what isn't. `tldr sh`, `tldr dash` and `tldr bash` also do it this way.

Reply:
I can assume when there is no "and then exit" command exists automatically and otherwise behavior must be explicitly defined. What about this?